system_tests/vision.py

```diff
@@ -190,6 +190,7 @@ def _assert_face(self, face):
 
     def test_detect_faces_content(self):
         client = Config.CLIENT
+        client._use_gax = False
```

Comment:
These are temporary until all the annotation types are covered in both JSON and protobuf.
This way the system tests pass when you run with gRPC enabled and disabled.
This just essentially forces the tests that don't support protobuf yet to always run in HTTP mode.